docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:centos6
```

Comment:
Maybe this could be more precise and point to a CentOS 6 particular version? E.g. 6.3, 6.4, 6.5.